src/secure_random.cr

```diff
@@ -19,4 +29,16 @@ module SecureRandom
     end
     slice
   end
+
+  def self.uuid
+    bytes = random_bytes(16)
+    {
+      bytes[0, 4].hexstring,
+      bytes[4, 2].hexstring,
+      bytes[6, 2].hexstring,
+      bytes[8, 2].hexstring,
+      bytes[10, 2].hexstring,
+      bytes[12, 4].hexstring,
+    }.join("-")
```

Comment:
So on all of these, the std dev is _huge_ (±673219.84, ±108942.84, etc) which makes either one coming ahead in the margin of error.